test/integration/features/cmd-config.feature

```diff
@@ -210,17 +210,8 @@ user defined options which changes default behaviour of Minishift.
      Then stdout should match "Disk size\s*:\s*25 GB"
      Then stdout should match "vCPUs\s*:\s*3"
       And Minishift should have state "Running"
-
-  Scenario: Checking that disk-size value was applied
-     When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/.da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
-
-  Scenario: Checking that cpus value was applied
-     When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
-     Then stdout should match "processor\s*: 0"
-      And stdout should match "processor\s*: 1"
-      And stdout should match "processor\s*: 2"
-      And stdout should not match "processor\s*: [3-9]"
+      And the vm should run with "3" cpus
+      And the vm should run with "25g" of disk size
```

Comment:
Better to use `And the vm should run with "25" GB of disk size`, that way it is documented and clear that it expects numbers in GBs + we do not need to parse it really, just Atoi() or maybe Godog takes care of it and the parameter can be directly integer.

Reply:
Right, this will avoid extra coding overhead

Reply:
Done, i use ```And the vm should run within "25" to "27" GB of disk size``` to make range visible outside as it depends on hypervisor.